src/shared/components/QueryBuilder/infrastructure/operatorsRepository.ts

```diff
@@ -3,13 +3,13 @@ import { Suggestions } from '../domain/types';
 class OperatorsRepository {
   async list(): Promise<Suggestions> {
     return [
-      { value: '=', label: '=', description: 'Equals' },
-      { value: '!=', label: '!=', description: 'Not equal' },
+      { value: '=', label: '=' },
```

Comment:
The list takes a lot of height, IMO no need to provide a description for trivial operators